bottles/frontend/main.py

```diff
@@ -336,6 +343,15 @@ def __show_about_window(self, *_args):
         about_window.set_transient_for(self.win)
         about_window.present()
 
+    def __show_library_view(self, *_args):
+        self.win.stack_main.set_visible_child_name("page_library")
+
+    def __show_bottles_view(self, *_args):
+        self.win.stack_main.set_visible_child_name("page_list")
+
+    def __go_back(self, *_args):
+        self.win.main_leaf.navigate(Adw.NavigationDirection.BACK)
```

Comment:
I tested it locally and it doesn't work well in practice. Instead of going back a page, it goes to the main page when I press Alt+Left:

https://github.com/bottlesdevs/Bottles/assets/50847364/b523a973-7b57-4b0b-ad08-27ac83271655

Reply:
I noted this too, thats why i didn't add this once to the keyboard shortcuts window, the reason for this being we have two separate leaflets, this should be fixed automatically when we switch to AdwNavigationView.